app/controllers/mentors_controller.rb

```diff
@@ -20,8 +20,17 @@ def mentor_params
     end
 
     def all_mentors_data(mentors)
-      mentors.each do |mentor|
-        { mentor_id: mentor.id, user_id: mentor.user_id, pic: mentor.user.pic, name: mentor.user.name, mentor_bio: mentor.bio, mentor_skill: mentor.skill }
-      end
+      mentors.map { |mentor| mentor_data(mentor) }
+    end
+
+    def mentor_data(mentor)
+      {
+        mentor_id: mentor.id,
+        user_id: mentor.user_id,
+        name: mentor.user.name,
+        pic: mentor.user.pic,
+        bio: mentor.bio,
+        skill: mentor.skill
+      }
```

Comment:
Would it be possible to have email in here too? Just so we can keep the super basic contact button on the mentor profile. If we can implement some kind of 'book' button today we should be okay without an email here though.

Reply:
Updated!